app/src/main/java/com/example/umlandowallet/ui/SettingsScreen.kt

```diff
@@ -197,5 +206,51 @@ fun SettingsScreen(navController: NavController) {
                 tint = Color(0xff000000)
             )
         }
+        Button(
+            onClick = {
+                println("New bitcoin address: ${Global.wallet!!.getAddress(AddressIndex.NEW).address}")
+            },
+            colors = ButtonDefaults.buttonColors(containerColor = Color(0xfff0f0f0)),
+            shape = RoundedCornerShape(8.dp),
+            modifier = Modifier
+                .size(width = 400.dp, height = 70.dp)
+                .padding(start = 24.dp, end = 24.dp, top = 24.dp)
+        ) {
+            Text(
+                text = "Get new address",
+                fontWeight = FontWeight.Normal,
+                color = Color(0xff2f2f2f)
+
+            )
+            Spacer(modifier = Modifier.weight(1f))
+            Icon(
+                painter = painterResource(id = R.drawable.ic_hicon_right_arrow),
+                contentDescription = "Right arrow icon",
+                tint = Color(0xff000000)
+            )
+        }
+        Button(
+            onClick = {
+                println("On chain balance: ${Global.wallet!!.getBalance()}")
+            },
+            colors = ButtonDefaults.buttonColors(containerColor = Color(0xfff0f0f0)),
+            shape = RoundedCornerShape(8.dp),
+            modifier = Modifier
+                .size(width = 400.dp, height = 70.dp)
+                .padding(start = 24.dp, end = 24.dp, top = 24.dp)
+        ) {
+            Text(
+                text = "Get new address",
```

Comment:
The label on this button is "Get new address" but it actually prints the balance.